web/src/connectors/edit-job-connector/edit-job-connector.tsx

```diff
@@ -116,6 +121,13 @@ const EditJobConnector: FC<EditJobConnectorProps> = ({
             };
         }
 
+        if (validationType === 'validation only') {
+            metadata.props['validators'] = {
+                validationMessage: `For Validation only at least 1 validator required`,
```

Comment:
I propose to use such form "For "Validation only" type at least 1 validator is required".